lib/src/windows/file_picker_windows.dart

```diff
@@ -94,78 +95,94 @@ class FilePickerWindows extends FilePicker {
     String? dialogTitle,
     bool lockParentWindow = false,
     String? initialDirectory,
-  }) {
+  }) async {
+    return compute(_getDirectoryPathIsolate, {
+      'dialogTitle': dialogTitle,
+      'initialDirectory': initialDirectory,
+    });
+  }
+
+  String? _getDirectoryPathIsolate(Map<String, dynamic> args) {
+    String? dialogTitle = args['dialogTitle'];
+    String? initialDirectory = args['initialDirectory'];
+
     int hr = CoInitializeEx(
       nullptr,
       COINIT.COINIT_APARTMENTTHREADED | COINIT.COINIT_DISABLE_OLE1DDE,
     );
 
-    // Ignore the error if COM is already initialized.
-    // If this is the case, CoInitializeEx will return
-    // RPC_E_CHANGED_MODE.
-    if (!SUCCEEDED(hr) && hr != RPC_E_CHANGED_MODE) {
```

Comment:
Just to confirm, the `hr != RPC_E_CHANGED_MODE` check is no longer needed, since we init COM in its own Isolate now?

Reply:
Yes the conflict was coming it was already initialized on the calling thread, with a new thread we don’t need this anymore.